internal/tiltfile/tiltfile_test.go

```diff
@@ -739,20 +761,9 @@ docker_build('gcr.io/c', 'c')
 `)
 	f.load()
 
-	f.assertNextManifest("a", deployment("deployment-a"))
-	f.assertNextManifest("b", deployment("deployment-b"), service("service-b"))
-	f.assertNextManifest("c", deployment("deployment-c"), service("service-c1"), service("service-c2"))
-}
-
-func TestK8sResourceWithoutDockerBuild(t *testing.T) {
```

Comment:
this seems like an important test? why did this get deleted?

Reply:
unsure. readded.